server/game/GameActions/PutIntoPlayAction.js

```diff
@@ -48,9 +48,11 @@ class PutIntoPlayAction extends CardGameAction {
             let choices = ['Left', 'Right'];
 
             if (
-                context.ability &&
-                context.ability.isCardPlayed() &&
-                (this.deploy || card.hasKeyword('deploy')) &&
+                ((context.ability && context.ability.isCardPlayed()) ||
+                    player.anyEffect('creaturesEnterPlayAnywhere', context)) &&
```

Comment:
This feels specific. I had a look to see if I could replace it with some check of the event `"onMakeToken"` instead (to be more generic), but I could not figure it out.

For now it works.